omnibus-mainnet.toml

```
@@ -7,20 +7,24 @@ include = [
     "tomls/core.toml",
     "tomls/permissions.toml",
     "tomls/collaterals/snx.toml",
+    "tomls/markets/legacy.toml",
     # "tomls/oracles/eth.toml",
     # "tomls/collaterals/weth.toml",
     "tomls/pools/spartan-council.toml",
     "tomls/permit-all-transferCrossChain.toml",
 ]
 
 [setting.snx_package]
-defaultValue = "synthetix:3.3.5"
+defaultValue = "synthetix:3.3.15"
 
 [setting.spot_market_package]
-defaultValue = "synthetix-spot-market:3.3.5"
+defaultValue = "synthetix-spot-market:3.3.15"
 
 [setting.perps_market_package]
-defaultValue = "synthetix-perps-market:3.3.5"
+defaultValue = "synthetix-perps-market:3.3.15"
```

1. go with the latest?
```suggestion
defaultValue = "synthetix-perps-market:3.3.19"
```
2. we can actually remove this as we dont have perps deployed, or do we?

---

thats right we dont have any perps deployed